publiq-ui/pub-pagination.vue

```diff
@@ -41,24 +41,24 @@
   .pagination {
     justify-content: center;
 
-    ::v-deep .page-item.active .page-link {
+    /deep/ .page-item.active .page-link {
       z-index: 0;
     }
 
-    ::v-deep .prev-btn .page-link {
+    /deep/ .prev-btn .page-link {
```

Comment:
Is this just a personal preference or is there a functional/technical difference?

Reply:
I'm okay with either but we should pick one and stick with it if there's no difference.

Reply:
I was planning to do a separate refactor for al occurrence into `/deep/`. it does exactly the same but my IDE picks up better on the syntax

Reply:
Get a better IDE maybe haha 😉 
But okay for me to replace them 👍